src/main/java/eu/europeana/metis/sandbox/common/amqp/RecordMessageConverter.java

```diff
@@ -72,7 +72,7 @@ public Object fromMessage(Message message) {
     byte[] content = message.getBody();
     MessageProperties properties = message.getMessageProperties();
     String recordId = properties.getHeader(RECORD_ID);
-    String datasetId = properties.getHeader(DATASET_ID);
+    Integer datasetId = properties.getHeader(DATASET_ID);
```

Comment:
Good find! Well the best would be indeed Integer. But to keep the consistency with metis, because of the legacy ids, we should keep it as String I would say..

Reply:
Yes, since I am generating the value from the database I set it as an Integer, because it will not be anything else, but what are we winning with keeping it as a String? It might be confusing to think it could be a String, and since this is a new application why not going with the right type?

Reply:
Reverted to String